package.json

```diff
@@ -1,32 +1,38 @@
 {
-  "name": "laminas-ci-matrix-action",
-  "engines": {
-    "node": "^18"
-  },
-  "type": "commonjs",
-  "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.16.0",
-    "@typescript-eslint/parser": "^5.16.0",
-    "@types/node": "^18.6.4",
-    "eslint": "^8.11.0",
-    "eslint-config-incredible": "^2.4.2",
-    "eslint-import-resolver-typescript": "^2.7.0",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-node": "^11.1.0",
-    "source-map-loader": "^3.0.1",
-    "typescript": "~4.6.3",
-    "webpack": "^5.70.0",
-    "webpack-cli": "^4.9.2"
-  },
-  "scripts": {
-    "lint": "eslint src/ --ext .ts",
-    "lint-fix": "eslint src/ --ext .ts --fix",
-    "build": "tsc --build && webpack --mode=production"
-  },
-  "dependencies": {
-    "@actions/core": "^1.6.0",
-    "@cfworker/json-schema": "^1.12.2",
-    "@types/semver": "^7.3.9",
-    "semver": "^7.3.5"
-  }
+    "name": "laminas-ci-matrix-action",
+    "description": "Create a CI matrix for use in a GitHub Action",
```

Comment:
Btw, please don't reformat everything next time 😁

Reply:
Yeah I wouldn't normally, but the vast majority of repos have `scripts` at the top, and I may have gone a bit too far :(